tests/integration/CoreTest.php

```diff
@@ -12,13 +12,46 @@
  */
 class CoreTest extends UnishIntegrationTestCase
 {
+    public function testCoreRequirements()
+    {
+        $root = $this->webroot();
+        $options = [
+        'pipe' => null,
+        'ignore' => 'cron,http requests,update,update_core,trusted_host_patterns', // no network access when running in tests, so ignore these
+        // 'strict' => 0, // invoke from script: do not verify options
+        ];
+        // Verify that there are no severity 2 items in the status report
+        $this->drush('core-requirements', [], $options + ['severity' => '2']);
+        $output = $this->getOutput();
+        $this->assertEquals('', $output);
+
+        $this->drush('core-requirements', [], $options);
+        $loaded = $this->getOutputFromJSON();
+        // Pick a subset that are valid for D6/D7/D8.
+        $expected = [
+        // 'install_profile' => -1,
+        // 'node_access' => -1,
+        'php' => -1,
+        // 'php_extensions' => -1,
+        'php_memory_limit' => -1,
+        'php_register_globals' => -1,
+        'settings.php' => -1,
+        ];
+        foreach ($expected as $key => $value) {
+            if (isset($loaded->$key)) {
+                $this->assertEquals("{$key}={$value}", "{$key}=" . $loaded->$key->sid);
+            }
+        }
+    }
+
     public function testDrupalDirectory()
     {
         $root = $this->webroot();
         $sitewide = $this->drupalSitewideDirectory();
+
         $this->drush('drupal-directory', ['%files']);
         $output = $this->getOutput();
-        $this->assertEquals(Path::join($root, '/sites/dev/files'), $output);
+        $this->assertEquals(Path::join($root, '/sites/default/files'), $output);
```

Comment:
I did that on purpose so that the integration tests could re-use the fixture site created on a previous test run with UNISH_DIRTY.

Open for discussion.